src/FsAutoComplete/CodeFixes/RenameUnusedValue.fs

```diff
@@ -81,9 +81,18 @@ let fix (getParseResultsForFile: GetParseResultsForFile) =
           if mfv.IsMemberThisValue then
             return [ mkReplaceFix diagnostic.Range ]
           elif mfv.IsValue then
-            return
-              [ tryMkValueReplaceFix diagnostic.Range; tryMkPrefixFix diagnostic.Range ]
-              |> List.choose id
+            let symbolString =
```

Comment:
Interested of indexing into strings directly, please use the indexer syntax on the NamedText type that accepts Ranges and returns Results. That helps to be more safe and prevent off-by-one errors.

Reply:
Oh yes, that's much better. Thanks!